apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -49,15 +40,15 @@ const LAYOUT_DATA: {
     layoutDisplayName: 'Default',
     layoutTypename: 'content',
     layoutDescription: 'This is the most basic layout for your content.',
-    imageSource: '/assets/layout-card/default_layout_card.png',
+    imageSource: '/assets/layout-card/default_layout_card.webp',
```

Comment:
webp is a newer file type ah, i hope gsibs can load 😆. should be can la according to https://caniuse.com/webp

Reply:
Checked, GSIBs can load webp! All good